toolchain/check/cpp_overload_resolution.cpp

```diff
@@ -0,0 +1,111 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include "toolchain/check/cpp_overload_resolution.h"
+
+#include "clang/Sema/Overload.h"
+#include "clang/Sema/Sema.h"
+#include "toolchain/check/cpp_type_mapping.h"
+#include "toolchain/check/import_cpp.h"
+
+namespace Carbon::Check {
+
+auto PerformCppOverloadResolution(Context& context, SemIR::LocId loc_id,
+                                  SemIR::InstId callee_id,
+                                  llvm::ArrayRef<SemIR::InstId> arg_ids)
+    -> std::optional<SemIR::InstId> {
+  Diagnostics::AnnotationScope annotate_diagnostics(
+      &context.emitter(), [&](auto& builder) {
+        CARBON_DIAGNOSTIC(InCallToCppFunction, Note,
+                          "in call to a Cpp function here");
```

Comment:
```suggestion
                          "in call to Cpp function here");
```
We generally leave off articles in our diagnostics where it doesn't introduce ambiguity.

Reply:
Done.